frontend/ui/elements/page_turns.lua

```diff
@@ -122,6 +122,18 @@ local PageTurns = {
                     end,
                 })
             end,
+        },
+        {
+            text = _("Also invert document-related dialogs"),
+            checked_func = function()
+                return G_reader_settings:isTrue("invert_ui_layout_mirroring")
```

Comment:
This setting looks too generic when read standalone.
It depends on `inverse_reading_order`, so it could be named `inverse_reading_order_also_ui_layout` - but then it is managed standalone in BD.invert() (which I think should not be aware that this is related to inverse_reading_order or to document related widgets).
Can't really think about how this could be made better, but it gets me confused when re-reviewing this.